### PR TITLE
Feature/avo 3184 background colors

### PR DESCRIPTION
**Pull request by ian-emsens-shd**
Screenshot 2024-05-16 at 13 30 58
Screenshot 2024-05-16 at 13 45 51
Screenshot 2024-05-16 at 13 46 00

### CODE REVIEW

File: src/assignment/views/AssignmentCreate.tsx [hunk 1]
```diff
@@ -245,12 +248,35 @@ const AssignmentCreate: FunctionComponent<DefaultSecureRouteProps> = ({
 						}}
 					/>
 				),
+				// Only assignments get to pick colors
+				actions: (item, i) =>
+					item && (
+						<>
+							<ListSorterColor item={item} />
+							<ListSorterPosition item={item} i={i} />
```
Comment (bertyhell) [inline src/assignment/views/AssignmentCreate.tsx:256]: use "index" instead of "i"

File: src/assignment/views/AssignmentCreate.tsx [hunk 2]
```diff
@@ -245,12 +248,35 @@ const AssignmentCreate: FunctionComponent<DefaultSecureRouteProps> = ({
...
+							<ListSorterSlice item={item} />
+						</>
+					),
 			},
 			listSorterItem: {
 				onSlice: (item) => {
 					confirmSliceModal.setEntity(item);
 					confirmSliceModal.setOpen(true);
```
Comment (bertyhell) [inline src/assignment/views/AssignmentCreate.tsx:264]: OPTIONAL: setEntity and setOpen could get out of sync
you could also opt for only setEntity and infer the isOpen if the Entity is set: const isOpen = !!entity

File: src/assignment/views/AssignmentEdit.tsx [hunk 3]
```diff
@@ -630,12 +633,37 @@ const AssignmentEdit: FunctionComponent<AssignmentEditProps & UserProps> = ({
 						}}
 					/>
 				),
+				// Only assignments get to pick colors
+				actions: (item, i) =>
+					item && (
+						<>
+							<ListSorterColor item={item} />
+							<ListSorterPosition item={item} i={i} />
```
Comment (bertyhell) [inline src/assignment/views/AssignmentEdit.tsx:641]: index

File: src/shared/components/BlockList/BlockList.tsx [hunk 4]
```diff
@@ -81,7 +81,7 @@ const BlockList: FC<BlockListProps> = ({ blocks, config }) => {
 						onCollectionButtonClicked={config?.ZOEK?.onCollectionButtonClicked || noop}
 						educationLevelId={config?.ZOEK?.educationLevelId}
 					/>,
-					'alt'
+					(block as Avo.Assignment.Block).color || '#edeff2'
```
Comment (bertyhell) [inline src/shared/components/BlockList/BlockList.tsx:84]: use 
```suggestion
					(block as Avo.Assignment.Block).color || GET_ASSIGNMENT_GREY().value
```

**Review by bertyhell — APPROVED** [review]

File: src/assignment/views/AssignmentCreate.tsx (hunk 2 above)
Comment (ian-emsens-shd) [reply src/assignment/views/AssignmentCreate.tsx:264]: Won't change this because it's not touched by BaO

**Pull request merged**